packages/ui/src/ui/pages/query-tracker/QueryResults/ErrorTree/CollapsedString.tsx

```diff
@@ -0,0 +1,32 @@
+import React, {FC, useMemo} from 'react';
+import {Link} from '@gravity-ui/uikit';
+import cn from 'bem-cn-lite';
+import './CollapsedString.scss';
+import {useToggle} from 'react-use';
+
+type Props = {
+    value: string;
+    limit?: number;
+};
+
+const block = cn('yt-collapsed-string');
+
+export const CollapsedString: FC<Props> = ({value, limit = 200}) => {
```

Comment:
There is no any QT specific logic, so the component might be moved to src/components

Reply:
UP